janus_core/calculations/md.py

```diff
@@ -454,26 +454,26 @@ def _restart_file(self) -> str:
         )
 
     @staticmethod
-    def get_log_header() -> str:
+    def get_stats_header() -> str:
```

Comment:
Do we also want to provide `@property` accessors?

Reply:
is this addressed?

Reply:
Can we make this a property?

Of the options:

```
@staticmethod
@property
def _stats_header() -> str:

@property
@staticmethod
def _stats_header() -> str:
    
@property
def _stats_header(self) -> str:
```

If I try to access `self._stats_header`, first returns a property object, the second throws an error (`takes 0 positional arguments but 1 was given`), and the third works, but means we can't call `MolecularDynamics._stats_header`, which we use elsewhere.

Reply:
I may be missing something, but I think I'll leave it as beyond the scope of this anyway, since this was just relabelling, and potentially come back to it fir #124 or otherwise